main/src/cgeo/geocaching/connector/gc/PocketQueryListAdapter.java

```diff
@@ -14,13 +14,12 @@
 import android.widget.Button;
 import android.widget.TextView;
 
-import java.util.List;
 
```

Comment:
Just cosmetic: Can you also remove one of the empty lines?

Reply:
thanks for spotting, I'll remove one